geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SftBuilder.scala

```diff
@@ -179,7 +158,7 @@ class SftBuilder {
   /** Get the type spec string associated with this builder...doesn't include dtg info */
   def getSpec = {
     val entryLst = List(entries.mkString(SepEntry))
-    val options = List(splitPart, enabledIndexesPart).flatten
+    val options = List(enabledIndexesPart).flatten
```

Comment:
could we make it so that options is a method, and then the Accumulo one could just override options like
<code>override def options = super.options ++ List(splitPart).flatten</code>
instead of overriding getSpec